### ISSUE TITLE
Tetrahedral matrix documentation

### ISSUE THREAD

**Issue by ThomasWilshaw** [ethan-ou/camera-match#5]
Hi,

Firstly thanks for creating this library, it's incredibly useful. I was wondering if you could provide some documentation on how to use the results of the Tetrahedral matrix values? Are they the red, green, blue, magenta, cyan and yellow triplets to be used in a standard tetrahedral colour tool or something else? I've set up a simple script to match two test charts as below:

```
matrix = TetrahedralMatrix()
matrix.solve(scanner_pixels_data_array, xyz_pixels_data_array)
print(matrix.matrix)
```

but when I input the results into a tetrahedral node in Nuke I get results that are definitely wrong:
![image](https://github.com/user-attachments/assets/4ede3f0d-47fc-4cdc-953e-316b40c868ab)
![image](https://github.com/user-attachments/assets/48446584-1cd7-4143-9e7f-9408f1f64b8f)

Any help would be greatly appreciated. Also I'm assuming these can also be used in pipelines? Something like this:
```
pipeline= Pipeline([TetrahedralMatrix(), RBF()])
pipeline.solve(bmpcc_data, film_data)
matrix = pipeline.nodes[0]
rbf = pipeline.nodes[1]
print(matrix.matrix)
rbf.export_LUT(path="after_tet_scanner_WB_to_rec709_65.cube", size=65)
```

If so does the above do a tetrahydral match and then do a RBF match on top of that?

**Comment by ethan-ou** [comment]
I should probably put docs on this but try the following order for the tetra matrix: Red, Yellow, Green, Cyan, Blue, Magenta. Or it might be easier to switch the order so that it matches the DCTL. What do you think?

Also your pipeline with Tetra + RBF _should_ work but worst case you can try having two pipelines one after each other.

**Comment by ThomasWilshaw** [comment]
Thanks I'll give that a try :) All the tools I've seen have been ordered R, G, B, C, M, Y so that would proably make sense, I guess so long as it's documented it doesn't matter too much. They also usually have controls for the black and white points but i don't know if you could pull that out of your code too?

If I do two pipelines would it be something like this (pseudo code):

```
pipeline1= Pipeline([TetrahedralMatrix()])
pipeline1.solve(bmpcc_data, film_data)

pipeline2 = Pipeline(RBF())
pipeline2.solve(pipeline1, film_data)
```

**Comment by ThomasWilshaw** [comment]
Yep that order seems to have worked worked :) I've not got a perfect match but its' very close. I'm using every pixel in the image as a data point, is that too much or is more always better do you think?

**Comment by ethan-ou** [comment]
For the two pipelines you can just use the nodes themselves:

```python
matrix = TetrahedralMatrix()
matrix.solve(bmpcc_data, film_data)

# Apply the matrix to the existing data
new_bmpcc_data = matrix(bmpcc_data)

rbf = RBF()
rbf.solve(new_bmpcc_data, film_data)
```

As for the number of data points having a lot is fine. It'll likely work so long as the data is high quality and fits what you're trying to match.

**Comment by ThomasWilshaw** [comment]
Thanks that's really helpful, I'll give it a shot.

**Comment by ethan-ou** [comment]
In the latest version of camera-match TetrahedralMatrix now has the proper ordering:  R-G-B-C-M-Y so this shouldn't be an issue anymore.
